Tools/SwiftBrowser/Source/ContentView.swift

```diff
@@ -55,6 +55,67 @@ fileprivate struct ToolbarBackForwardMenuView: View {
     }
 }
 
+fileprivate struct DialogActionsView: View {
+    private let dialog: DialogPresenter.Dialog
+
+    @State private var promptText = ""
+
+    init(dialog: DialogPresenter.Dialog) {
+        self.dialog = dialog
+
+        if case let .prompt(_, defaultText, _) = dialog.configuration, let defaultText {
+            _promptText = State(initialValue: defaultText)
+        }
+    }
+
+    var body: some View {
+        switch dialog.configuration {
+        case let .alert(_, dismissAlert):
+            Button("Dismiss") {
+                dismissAlert()
+            }
+
+        case let .confirm(_, dismissConfirm):
+            Button("OK") {
+                dismissConfirm(.ok)
+            }
```

Comment:
Should `.keyboardShortcut(.defaultAction)` be applied here too?

Reply:
it's not needed, since if there are only buttons involved, SwiftUI can figure out which one is the default if the other is cancel.

Reply:
Yeah, if you had more than one non-cancel non-destructive role button you'd probably need that to tie break otherwise.